shell/src/main/java/alluxio/shell/command/FileInfoCommand.java

```diff
@@ -58,12 +57,34 @@ protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioExcepti
   }
 
   @Override
+  protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
+    URIStatus status = mFileSystem.getStatus(path);
+    if (status.isFolder()) {
+      List<URIStatus> childrens = mFileSystem.listStatus(path);
```

Comment:
I do not find a command that can list all children of a directory. So, I think we can add a option for user to choice whether listing all the children of the path.